docs/adr/005_observability.md

```diff
@@ -74,24 +74,17 @@ In addition to this, we will support Application Insights metrics on an opt-in b
 
 | Name                       | Description                                                  | Source | Namespace | Dimensions                 |
 | -------------------------- | ------------------------------------------------------------ | ------ | --------- | -------------------------- |
-| RxWndRate                  | Number of times we hit the different receive windows.        | LNS    | LoRaWan   | Gateway Id, Receive Window |
-| RxWndMiss                  | Number of missed on downstream windows                       | LNS    | LoRaWan   | Gateway Id                 |
+| ReceiveWindowHits          | Number of times we hit the different receive windows.        | LNS    | LoRaWan   | Gateway Id, (estimated) Receive Window |
+| ReceiveWindowMisses        | Number of missed on downstream windows                       | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceCacheHit             | Number of device cache hit                                   | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceLoadRequests         | Number of device load requests                               | LNS    | LoRaWan   | Gateway Id                 |
 | JoinRequests               | Number of join requests                                      | LNS    | LoRaWan   | Gateway Id                 |
 | StationConnectivityLost    | Connection to LBS lost                                       | LNS    | LoRaWan   | Gateway Id                 |
 | ActiveStationConnections   | Active connections to stations                               | LNS    | LoRaWan   | Gateway Id                 |
 | D2CMessagesReceived        | Number of messages received from device                      | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessageDeliveryLatency  | Time from when we received the message from the concentrator until we are done processing it | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessagesDelivered       | Number of messages sent to upstream                          | LNS    | LoRaWan   | Gateway Id, To             |
-| D2CMessagesError           | Number of errors in sending messages to upstream             | LNS    | LoRaWan   | Gateway Id, To            |
```

Comment:
HighUpstreamMessageErrorRatio and HighDownstreamMessageErrorRatio won't be possible without "error" metrics.

Reply:
I removed the error metrics as discussed with @p-schuler. Alerting on unhandled exceptions should be good enough, as in the LoRa case it's quite hard to distinguish whether errors occur due to downstream or upstream messages (the logic is interconnected due to the nature of LoRa with the receive windows for class A devices)

Reply:
Now I brought it back to also give users that do not use Azure Monitor insights around the number of Processing Errors.